mods/dispatcher/src/types.ts

```diff
@@ -42,7 +42,7 @@ export type CallbackErrors =
 
 export type ProcessorCallback = (
   err: CallbackErrors,
-  reponse?: MessageRequest
+  response?: MessageRequest
```

Comment:
Please review (may be breaking change)

Reply:
Hey @luzpaz 

Thanks for the note. 

I believe it will be ok since it's a positional parameter.